slave/osx/fabfile.py

```diff
@@ -0,0 +1,48 @@
+from fabric.api import sudo, task, settings, cd, put
+from twisted.python.filepath import FilePath
+
+
+@task
+def create_user(username, uid=650):
+    """
+    Create an OS X user.
+
+    UID must be > 500 for the user to show up in
+    /System/Library/PreferencePanes/Accounts.prefPane.
+
+    UID cannot conflict with an existing user.
+    UID and PrimaryGroupID were chosen to mimic a created standard user on a
+    test Mac.
+    """
+    sudo("""
+    dscl . -create /Users/%(username)s
+    dscl . -create /Users/%(username)s UserShell /bin/bash
+    dscl . -create /Users/%(username)s RealName "ClusterHQ Buildslave"
+    dscl . -create /Users/%(username)s UniqueID "%(uid)s"
+    dscl . -create /Users/%(username)s PrimaryGroupID 20
```

Comment:
Obviously, I'm blind, since it is mentioned in the docstring. Nevertheless, the name of the corresponding group should be given.